Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -6746,7 +6768,12 @@ ElementInfo IXPath.GetElementParent(ElementInfo ElementInfo)
                 return parentEI;
             }
 
-            parentEI = GetElementInfoFromIWebElement(parentElementIWebElement, parentElementHtmlNode, ElementInfo);
+            Tuple<string, eElementType> elementTypeEnum = GetElementTypeEnum(htmlNode: parentElementHtmlNode);
```

Comment:
no need to find type here, let it be found inside the FindeDetails() func

Reply:
Done ! Logic Simplified !